Wrappers/Python/cil/optimisation/algorithms/Algorithm.py

```diff
@@ -25,31 +25,21 @@
 
 
 class Algorithm:
-    '''Base class for iterative algorithms
+    r"""Base class for iterative algorithms that provides the minimal infrastructure.
```

Comment:
```suggestion
    r"""Base class providing minimal infrastructure for iterative algorithms.
```

Reply:
The proposal seeks to clarify that it is the base class, and not the iterative algorithms, that provide the minimal infrastructure.

Reply:
Done